qstack/regression/hyperparameters.py

```diff
@@ -127,16 +127,25 @@ def main():
     parser.add_argument('--ada',  action='store_true', dest='adaptive', default=False,  help='if adapt sigma')
     parser.add_argument('--readkernel', action='store_true', dest='readk', default=False,  help='if X is kernel')
     parser.add_argument('--sparse',     type=int, dest='sparse', default=None,  help='regression basis size for sparse learning')
+    parser.add_argument('--name',      type=str,   dest='nameout',       required=False, default=None,  help='the name of the output file')
+    parser.add_argument('--select',      type=str,   dest='f_select',       required=False, help='a txt file containing the indices of the selected representations')
```

Comment:
Yes, they can if they use the function directly but not if they use `main()`
Otherwise they need to regenerated the `.npy` arrays (passed to CLI) with the restricted selection.

This way you just have one `.npy` array and then as many `.txt` lists (containing indices) as you want to try out various selection.

Reply:
Actually, now that I had to use it I remember, the main point of this option is to be able to restrict the initial dataset. Typically, if I want to truncate or slice the initial dataset according to some threshold on the property (to change/smoothen its distribution) I just list the indices instead of producing a new array of representations AND a new property file. But then the rest should run normally/random.
_Is it too far fetch? maybe this type of stuff should be dealt by some home-made script calling the function instead?_

Reply:
Also, though 
> then it would make sense to have txt files for train/validation/test, what do you think?    

this would make sense for hyperparameter and regression, it wouldn't make sense for cross_validate_results, because then you would need to specify train/val/test sets for all repetitions (normally over seeds), since the point is to vary the test set.

Reply:
> Is it too far fetch? maybe this type of stuff should be dealt by some home-made script calling the function instead?

Reply:
> this would make sense for hyperparameter and regression, it wouldn't make sense for cross_validate_results

Reply:
Then I will remove the `--select` option from all three scripts.
And I will add a `--selection` option and document it in the `parser` help!